B/booster/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "booster"
+version = v"0.1.2"
+
+# Collection of sources required to build SFML
+sources = [
+    GitSource("https://github.com/evolbioinfo/booster.git", "a5f5044b192db7ab5572f44ef46abaea007f5ce9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/booster
+install_license ${WORKSPACE}/srcdir/booster/LICENSE
+
+# compile booster
+cd ${WORKSPACE}/srcdir/booster/src
+make -j${nproc}
```

Comment:
I don't really like their default choices.  We want to use clang on BSD systems:
```suggestion
make -j${nproc} CC=${CC} CFLAGS_OMP="-Wall -g -fopenmp"
```